### PR TITLE
[Improve][CDC]Remove  driver for cdc connector

### PR DESCRIPTION
**Pull request by liugddx**
## Purpose of this pull request

- remove driver dependencies for cdc connector
- Change log level to warn

## Check list

* [ ] Code changed are covered with tests, or it does not need tests for reason:
* [ ] If any new Jar binary package adding in your PR, please add License Notice according
  [New License Guide](https://github.com/apache/seatunnel/blob/dev/docs/en/contribution/new-license.md)
* [ ] If necessary, please update the documentation to describe the new feature. https://github.com/apache/seatunnel/tree/dev/docs
* [ ] If you are contributing the connector code, please check that the following files are updated:
  1. Update change log that in connector document. For more details you can refer to [connector-v2](https://github.com/apache/seatunnel/tree/dev/docs/en/connector-v2)
  2. Update [plugin-mapping.properties](https://github.com/apache/seatunnel/blob/dev/plugin-mapping.properties) and add new connector information in it
  3. Update the pom file of [seatunnel-dist](https://github.com/apache/seatunnel/blob/dev/seatunnel-dist/pom.xml)
* [ ] Update the [`release-note`](https://github.com/apache/seatunnel/blob/dev/release-note.md).

### CODE REVIEW

**Comment by ocean-zhc** [comment]
这个提示 没有驱动
```
Caused by: java.lang.RuntimeException: Failed to get driver instance for jdbcUrl=jdbc:mysql://localhost:3306/test
        at com.zaxxer.hikari.util.DriverDataSource.<init>(DriverDataSource.java:114)
        at com.zaxxer.hikari.pool.PoolBase.initializeDataSource(PoolBase.java:331)
        at com.zaxxer.hikari.pool.PoolBase.<init>(PoolBase.java:114)
        at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:108)
        at com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:81)
        at org.apache.seatunnel.connectors.cdc.base.relational.connection.JdbcConnectionPoolFactory.createPooledDataSource(JdbcConnectionPoolFactory.java:52)
        at org.apache.seatunnel.connectors.cdc.base.relational.connection.JdbcConnectionPools.getOrCreateConnectionPool(JdbcConnectionPools.java:56)
        at org.apache.seatunnel.connectors.cdc.base.relational.connection.JdbcConnectionFactory.connect(JdbcConnectionFactory.java:60)
        at io.debezium.jdbc.JdbcConnection.connection(JdbcConnection.java:890)
        at io.debezium.jdbc.JdbcConnection.connection(JdbcConnection.java:885)
        at io.debezium.jdbc.JdbcConnection.connect(JdbcConnection.java:418)
        at org.apache.seatunnel.connectors.cdc.base.dialect.JdbcDataSourceDialect.openJdbcConnection(JdbcDataSourceDialect.java:52)
        ... 9 more
Caused by: java.sql.SQLException: No suitable driver
        at java.sql.DriverManager.getDriver(DriverManager.java:315)
        at com.zaxxer.hikari.util.DriverDataSource.<init>(DriverDataSource.java:106)
        ... 20 more

        ... 11 more

```

**Comment by liugddx** [comment]
> 这个提示 没有驱动
> 
> ```
> Caused by: java.lang.RuntimeException: Failed to get driver instance for jdbcUrl=jdbc:mysql://localhost:3306/test
>         at com.zaxxer.hikari.util.DriverDataSource.<init>(DriverDataSource.java:114)
>         at com.zaxxer.hikari.pool.PoolBase.initializeDataSource(PoolBase.java:331)
>         at com.zaxxer.hikari.pool.PoolBase.<init>(PoolBase.java:114)
>         at com.zaxxer.hikari.pool.HikariPool.<init>(HikariPool.java:108)
>         at com.zaxxer.hikari.HikariDataSource.<init>(HikariDataSource.java:81)
>         at org.apache.seatunnel.connectors.cdc.base.relational.connection.JdbcConnectionPoolFactory.createPooledDataSource(JdbcConnectionPoolFactory.java:52)
>         at org.apache.seatunnel.connectors.cdc.base.relational.connection.JdbcConnectionPools.getOrCreateConnectionPool(JdbcConnectionPools.java:56)
>         at org.apache.seatunnel.connectors.cdc.base.relational.connection.JdbcConnectionFactory.connect(JdbcConnectionFactory.java:60)
>         at io.debezium.jdbc.JdbcConnection.connection(JdbcConnection.java:890)
>         at io.debezium.jdbc.JdbcConnection.connection(JdbcConnection.java:885)
>         at io.debezium.jdbc.JdbcConnection.connect(JdbcConnection.java:418)
>         at org.apache.seatunnel.connectors.cdc.base.dialect.JdbcDataSourceDialect.openJdbcConnection(JdbcDataSourceDialect.java:52)
>         ... 9 more
> Caused by: java.sql.SQLException: No suitable driver
>         at java.sql.DriverManager.getDriver(DriverManager.java:315)
>         at com.zaxxer.hikari.util.DriverDataSource.<init>(DriverDataSource.java:106)
>         ... 20 more
> 
>         ... 11 more
> ```

You can test with dev branch

**Comment by liugddx** [comment]
@EricJoy2048 @hailin0 @ic4y @Hisoka-X PTAL.

File: seatunnel-connectors-v2/connector-jdbc/pom.xml [hunk 1]
```diff
@@ -30,7 +30,7 @@
     <name>SeaTunnel : Connectors V2 : Jdbc</name>
 
     <properties>
-        <mysql.version>8.0.16</mysql.version>
+        <mysql.version>8.0.27</mysql.version>
```
Comment (ic4y) [inline seatunnel-connectors-v2/connector-jdbc/pom.xml:33]: What is the reason for changing this version

Comment (liugddx) [reply seatunnel-connectors-v2/connector-jdbc/pom.xml:33]: https://debezium.io/releases/1.6/

debezium using 8.0.27

Comment (liugddx) [reply seatunnel-connectors-v2/connector-jdbc/pom.xml:33]: https://issues.redhat.com/plugins/servlet/mobile#issue/DBZ-4286

**Review by ic4y — APPROVED** [review]

**Review by EricJoy2048 — APPROVED** [review]

**Pull request merged**